data/org.freedesktop.portal.RemoteDesktop.xml

```diff
@@ -34,13 +34,15 @@
         Create a remote desktop session.
 
         A remote desktop session is used to allow remote controlling a desktop
-        session. It can also be used together with a screen cast session (see
-        #org.freedesktop.portal.ScreenCast), but may only be started and stopped
-        with this interface.
+        session.
 
-        To also get a screen content, call the
-        org.freedesktop.portal.ScreenCast.SelectSources() with the
-        #org.freedesktop.portal.Session object created with this method.
+        A remote desktop session may only be started and stopped with this interface,
+        but you can use the #org.freedesktop.portal.Session object created with this
+        method together with certain methods on the #org.freedesktop.portal.ScreenCast
+        interface. Specifically, you can call
+        org.freedesktop.portal.ScreenCast.SelectSources() to also get screen content,
+        or org.freedesktop.portal.ScreenCast.OpenPipewireRemote() to acquire a file
```

Comment:
`or` → `and`

I think it's worth adding something like `See #org.freedesktop.portal.ScreenCast for more information on how to operate it`, because one cannot call OpenPipeWireRemote() without calling SelectSources() first.